File: main/src/main/java/de/blinkt/openvpn/activities/LogWindow.java
package de.blinkt.openvpn.activities;

import android.app.Activity;
import android.os.Bundle;
import android.view.MenuItem;

import de.blinkt.openvpn.R;
import de.blinkt.openvpn.fragments.LogFragment;

/**
 * Created by arne on 13.10.13.
 */
public class LogWindow extends Activity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.log_window);
        getActionBar().setDisplayHomeAsUpEnabled(true);

        if (savedInstanceState == null) {
            getFragmentManager().beginTransaction()
                    .add(R.id.container, new LogFragment())
                    .commit();
        }

    }

    @Override
    public boolean onOptionsItemSelected(MenuItem item) {
        return super.onOptionsItemSelected(item);
    }
}

File: main/src/ui/java/de/blinkt/openvpn/activities/LogWindow.kt
/*
 * Copyright (c) 2012-2016 Arne Schwabe
 * Distributed under the GNU GPL v2 with additional terms. For full terms see the file doc/LICENSE.txt
 */
package de.blinkt.openvpn.activities

import android.os.Bundle
import android.view.MenuItem
import de.blinkt.openvpn.R
import de.blinkt.openvpn.fragments.LogFragment

/**
 * Created by arne on 13.10.13.setUpEdgeEdgeStuff
 */
class LogWindow : BaseActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.log_window)
        supportActionBar?.setDisplayHomeAsUpEnabled(true)

        if (savedInstanceState == null) {
            supportFragmentManager.beginTransaction()
                .add(R.id.container, LogFragment())
                .commit()
        }

        setUpEdgeEdgeInsetsListener(getWindow().getDecorView().getRootView(), R.id.container)
    }

    override fun onOptionsItemSelected(item: MenuItem): Boolean {
        return super.onOptionsItemSelected(item)
    }
}
